Source/ablastr/fields/PoissonSolver.H

```diff
@@ -280,10 +285,13 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
             const amrex::IntVect& refratio = rel_ref_ratio.value()[lev];
             ba.coarsen(refratio);
             const int ncomp = linop.getNComp();
-            amrex::MultiFab phi_cp(ba, phi[lev+1]->DistributionMap(), ncomp, 1);
+            const int ng = (WarpX::grid_type == GridType::Collocated) ? 1 : 0;
+            amrex::MultiFab phi_cp(ba, phi[lev+1]->DistributionMap(), ncomp, ng);
+            if (ng > 0) {
+                phi_cp.setDomainBndry(0.0_rt, geom[lev]);
+            }
```

Comment:
Adding a comment (copied from AMReX's docstring for `setDomainBndry`):
```suggestion
            if (ng > 0) {
                // Set all values outside the domain to zero
                phi_cp.setDomainBndry(0.0_rt, geom[lev]);
            }
```